src/shared/components/palette/dynamic-radio-list/write-dynamic-radio-list-field.component.spec.ts

```diff
@@ -0,0 +1,97 @@
+import { async, ComponentFixture, TestBed } from '@angular/core/testing';
+import { FormGroup, ReactiveFormsModule } from '@angular/forms';
+import { DebugElement } from '@angular/core';
+import { FieldType } from '../../../domain/definition/field-type.model';
+import { CaseField } from '../../../domain/definition/case-field.model';
+import { PaletteUtilsModule } from '../utils/utils.module';
+import { By } from '@angular/platform-browser';
+import { WriteDynamicRadioListFieldComponent } from './write-dynamic-radio-list-field.component';
+import { attr, text } from '../../../test/helpers';
+
+const VALUE = 'F';
+const EXPECTED_LABEL = 'Female';
+const FIELD_ID = 'MarritalStatus';
```

Comment:
_Nitpick:_
`'Marrital'`? I swear I reviewed this before and it got changed... did something get lost along the way?

Reply:
Updated spelling as that annoyed me